src/site/content/en/authors/index.njk

```diff
@@ -16,7 +16,6 @@ pagination:
 <div class="authors-landing-page">
   {# Make sure Actions are first in the tab order. #}
   <web-actions actions="subscribe"></web-actions>
-
   <div class="w-layout-container w-pb--non">
```

Comment:
I think these are just whitespace changes. Might be able to fix them to cut down on noise in the PR. Otherwise they'll show up in git blame.